### PR TITLE
chore: workspace inheritance

### PR DESCRIPTION
**Pull request by yihau**
#### Problem

Some crates don't use workspace inheritance

#### Summary of Changes

make them use it ⛏️

### CODE REVIEW

File: programs/sbf/rust/big_mod_exp/Cargo.toml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
 
 [dependencies]
-array-bytes = "=1.4.1"
-serde = { version = "1.0.112", features = ["derive"] }
-serde_json = "1.0.56"
```
Comment (t-nelson) [inline programs/sbf/rust/big_mod_exp/Cargo.toml:15]: both of these will be updated by moving to the workspace version. probably not a problem, just wanted to raise

**Review by t-nelson — APPROVED** [review]
lgtm!

**Review by CriesofCarrots — APPROVED** [review]

**Comment by CriesofCarrots** [comment]
Whoa, TIL you can set automerge before you actually receive review approval.

**Comment by t-nelson** [comment]
> Whoa, TIL you can set automerge before you actually receive review approval.

same!  got scared for a second that i'd merged it by accident :sweat_smile:

**Comment by yihau** [comment]
can't remember who is the first one use this but it is really useful 😈

**Pull request merged**